### DISCUSSION TITLE
Can't fetch audio buffer

### DISCUSSION THREAD

**Discussion by MazaadyPortal** [ant-media/Ant-Media-Server#6108] [Announcements]
### Short description
can't fetch audio buffer for using it to show sound waves so please let me know how the callback name for doing that?

### Environment

iOS

**Comment by USAMAWIZARD** [comment]
let me check this and get back to you.

**Reply by MazaadyPortal** [reply]
any updates?

**Reply by USAMAWIZARD** [reply]
@MazaadyPortal , I have discussed this internally with our developers , we will need some more information to be able to assist you. can you please describe requirements in  more details

**Comment by MazaadyPortal** [comment]
The seller starts a live audio broadcast with interactive waves, and buyers can watch the broadcast, but I ran into a problem, which is that I need an audio buffer in order to be able to draw the seller’s interactive waves so that the buyer can see them.
I need to know if there is a function through which I can handle the audio (usually known as buffer) in your SDK?
________________________________

**Reply by USAMAWIZARD** [reply]
correct me if I am wrong you would like to get the audio level of the microphone of the seller ?
